frontend/packages/lib-components/src/white-space.ts

```diff
@@ -47,10 +47,16 @@ export const defaultMargins: Record<SpacingSize, string> = {
 type GapProps = {
   horizontal?: boolean
   size?: SpacingSize
+  hideOnMobile?: boolean
```

Comment:
I'm wondering if `sizeMobile?: SpacingSize` would be more useful, given that `'zero'` is a valid `SpacingSize` and sometimes you want the gap to be smaller/bigger on mobile, not just to hide it.